README.md

```diff
@@ -206,7 +206,11 @@ To deploy to the **production** server, replace `ukraine_dev` with `ukraine_prod
 
 The administration panel, which currently doesn't do a whole lot, can be accessed in development at http://localhost:3000/admin/ and in production at https://theyvoteforyou.org.au/admin. You must be an admin to be able to access that page. Any user that is an admin can make another user and admin too using the admin panel. The first admin user must be created via the rails console:
 ```
-$ bundle exec rails c
+deploy@hostname:/srv/www/production/current$ RAILS_ENV=production bundle exec rails c
+Loading production environment (Rails 8.0.2)
```

Comment:
BTW there's an email address in here. Are you sure about putting this in the README?